package.json

```diff
@@ -44,6 +45,8 @@
   },
   "dependencies": {
     "atlassian-jwt": "^1.0.3",
-    "axios": "^0.21.0"
+    "axios": "^0.21.0",
+    "oauth": "^0.9.15",
+    "qs": "^6.9.4"
```

Comment:
I think we should remove this dependency and create own implementation

Reply:
you're right. I'll look into this during the holidays.